clouddriver-titus/src/main/groovy/com/netflix/spinnaker/clouddriver/titus/client/TitusClientAuthenticationUtil.java

```diff
@@ -33,7 +37,21 @@
   public static <STUB extends AbstractStub<STUB>> STUB attachCaller(STUB serviceStub) {
     Metadata metadata = new Metadata();
     metadata.put(CALLER_ID_KEY, AuthenticatedRequest.getSpinnakerUser().orElse("spinnaker"));
-    metadata.put(CALL_REASON_KEY, AuthenticatedRequest.getSpinnakerExecutionId().orElse("unknown"));
+    metadata.put(
+        CALL_REASON_KEY,
+        String.format(
+            "Invoked by Spinnaker execution %s, Application name %s",
+            getSpinnakerExecutionId(
+                AuthenticatedRequest.getSpinnakerExecutionId().orElse("unknown")),
```

Comment:
nit: sort of defeats the purpose of `nullables` i suppose, but i think this would be more readable as `orElse(null)`

then `getSpinnakerExecutionId` below would read simpler as 
```
if (Strings.isNullOrEmpty(executionIdHeader) {
  return "unknown";
}

return Splitter.on(':').splitToList(executionIdHeader).get(0);
```

Reply:
never mind my comment as i see you followed, Robs :)